akvo/rsr/models/project.py

```diff
@@ -1469,3 +1469,16 @@ def update_denormalized_project(sender, **kwargs):
     project = project_update.project
     project.last_update = project_update
     project.save(last_updated=True)
+
+
+@receiver(post_delete, sender=ProjectUpdate)
+def rewind_last_update(sender, **kwargs):
+    """ Updates the denormalized project.last_update on related project,
```

Comment:
As with the test doc-string, there may be other tools in the eco-system which assume the first line is a summary line.